book/website/collaboration/inclusive-events/inclusive-events-comms.md

```diff
@@ -0,0 +1,487 @@
+to duplicate the communication of core information and engage with the communities of people with lived experience to understand how to communicate best with them. 
+
+You can find more information about other specific content considerations in a [blog post](https://accessibility.blog.gov.uk/2016/09/02/dos-and-donts-on-designing-for-accessibility/) 
+published by the Accessibility Team of the UK's Home Office Digital Group. 
+These posters make recommendations for how to make communications and information accessible for people with: 
+- low vision
+- D/deaf and hard of hearing
+- dyslexia
+- motor disabilities
+- users on the autistic spectrum
+- users of screen readers
+
+(cl-inclusive-events-comms-vendor-timelines)=
+### Working with Vendors 
+As part of deciding what event and promotional materials will be required, identify who will develop these materials, whether it’s an external marketing agency or 
+an internal graphic designer, and if there are additional requirements that vendors may have, such as specific printers, lead times, and file format preferences. 
+
+This is especially important for in-person or hybrid events where you may be printing posters, flyers, or program books for the event. 
+
+Vendors may not be able to print in certain colours which can undermine the efforts you have made to align with accessibility guidance. 
+
+By checking this with a supplier, you will be able to adjust designs and content if anything goes wrong. 
+
+(cl-inclusive-events-comms-accessibility-hashtags)=
+### Using Hashtags
+If you plan on developing a hashtag for your event, consider how you intend attendees to interact with the hashtag and for what purpose. 
+A hashtag that aims to raise awareness of your event will be different from one that raises awareness of your organization and research area. 
+Event specific hashtags can be useful to assess online engagement at your event, and can also be used to develop a greater awareness of your event if social media plays a large role in attracting attendees. 
+
+If you do choose to use a hashtag, try using Pascal Case (sometimes called Title Case) #WhichLooksLikeThis, where the first letter in each word is capitalized. 
+Using Lowercase and uppercase letters can benefit people with dyslexia, and can also help a screen reader identify separate words, allowing the technology to pronounce the hashtag correctly. 
+
+Try to avoid using a hyphen in your hashtags which some social media sites will read as two separate words. 
+If you can't keep your hashtag to a single word or phrase, use an underscore instead #Which_Looks_Like_This.
+
+(cl-inclusive-events-comms-key-information)=
+## Key Information Checklist
+This is not an exhaustive list of what to include in core communication, however, it can be used as a reference in addition to the information above for what to include on a website or in an information pack. 
```

Comment:
This again is a very detailed checklist that partially repeats points made above, but also introduces some new ones. Would it maybe be better to include the new points as individual points above (e.g. dress code) and then put the checklist as its own communications subsection at the end instead of merging it into the main body?

Reply:
I made dress code and sickness policy their own sections, and edited the checklist 🚀